lib/main_ltp.pm

```diff
@@ -31,9 +30,9 @@ use Utils::Backends 'is_pvm';
 # use warnings;
 
 our @EXPORT = qw(
-  get_ltp_tag
   load_kernel_tests
-  loadtest_from_runtest_file
+  loadtest
```

Comment:
Here's the problem. `main_ltp.pm` is imported from `main_common.pm` so exporting `loadtest` like this will override for non-LTP tests as well.